.github/scripts/build.sh

```diff
@@ -32,30 +32,36 @@ if [[ "$CCACHE" = 1 ]]; then
 fi
 
 $ROOT_PATH/.github/scripts/binutils/build.sh
+$ROOT_PATH/.github/scripts/clean-build.sh
```

Comment:
All the `clen-build.sh` scripts should be launched only when  `"$DELETE_BUILD" = 1`.